src/components/cc-zone-card/cc-zone-card.js

```diff
@@ -0,0 +1,213 @@
+import { css, html, LitElement } from 'lit';
+import { iconRemixCheckboxCircleFill as selectedIcon } from '../../assets/cc-remix.icons.js';
+import { skeletonStyles } from '../../styles/skeleton.js';
+import { i18n } from '../../translations/translation.js';
+import '../cc-badge/cc-badge.js';
+import '../cc-icon/cc-icon.js';
+import '../cc-img/cc-img.js';
+
+/**
+ * @typedef {import('./cc-zone-card.types.js').ZoneImage} ZoneImage
+ */
+
+/**
+ * A component displaying a card with relative zone information (name, code...)
+ *
+ * @cssdisplay flex
+ */
+export class CcZoneCard extends LitElement {
+  static get properties() {
+    return {
+      code: { type: String },
+      countryCode: { type: String, attribute: 'country-code' },
+      country: { type: String },
+      disabled: { type: Boolean, reflect: true },
+      flagUrl: { type: String },
+      images: { type: Array },
+      infra: { type: String },
+      name: { type: String },
+      selected: { type: Boolean, reflect: true },
+    };
+  }
+
+  constructor() {
+    super();
+
+    /** @type {string} The zone code */
+    this.code = null;
+
+    /** @type {string} The country name */
+    this.country = null;
+
+    /** @type {string} The country code (e.g: FRA) */
```

Comment:
issue: country codes are ISO 3166-1 alpha-2, they have 2 letters. We should mention this and use a proper example.